questionnaires/covid19_symptoms/covid19_symptoms_armt.json

```diff
@@ -0,0 +1,866 @@
+        "form_name": "covid19_symptoms",
+        "section_header": "",
+        "field_type": "yesno",
+        "field_label": "Have you changed your day to day activities since the COVID-19 outbreak? \r\n\r\nFor example, working from home; having to look after children because of school closures; having to look after an ill relative",
+        "select_choices_or_calculations": "",
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "medications_intake",
+        "form_name": "covid19_symptoms",
+        "section_header": "",
+        "field_type": "yesno",
+        "field_label": "Are you taking any medication?\r\n\r\nEither for prior conditions or to treat symptoms of COVID19?",
+        "select_choices_or_calculations": "",
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "",
+        "evaluated_logic": ""
+    },
+    {
+        "field_name": "covid_meds_dlist",
+        "form_name": "covid19_symptoms",
+        "section_header": "",
+        "field_type": "checkbox",
+        "field_label": "Please select if you are taking one or more of these?",
+        "select_choices_or_calculations": [
+            {
+                "code": "1",
+                "label": "Ibuprofen "
+            },
+            {
+                "code": "2",
+                "label": "Dicolfenac "
+            },
+            {
+                "code": "3",
+                "label": "Vitamin D "
+            },
+            {
+                "code": "4",
+                "label": "Other"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "[medications_intake] = '1'",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "",
+        "evaluated_logic": "responses['medications_intake'] ==  '1'"
+    },
+    {
+        "field_name": "covid19_meds_other",
+        "form_name": "covid19_symptoms",
+        "section_header": "",
+        "field_type": "notes",
```

Comment:
@afolarin We don't support notes, only text box. Can I switch this to text box?

Reply:
Sorry I got the wrong type. Pls change